local-dev.yml

```diff
@@ -156,15 +151,96 @@ services:
       - ".:/hydroshare"
     links:
       - pgbouncer:pgbouncer
-      - rabbitmq:rabbitmq
+      - redpanda:redpanda
     depends_on:
       - hydroshare
       - pgbouncer
       - solr
-      - rabbitmq
+      - redpanda
     stdin_open: true
     tty: true
     command: /bin/bash init-defaultworker
+  hs_event_s3:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: s3eventworker
+    platform: linux/amd64
+    hostname: s3eventworker
+    environment:
+      POSTGIS_HOST: pgbouncer
+      POSTGIS_PORT: 6432
+      POSTGIS_PASSWORD: postgres
+      POSTGIS_DB: postgres
+      POSTGIS_USER: postgres
+      PGPASSWORD: postgres
+      C_FORCE_ROOT: 1
+      SOLR_PORT_8983_TCP_ADDR: solr
+      SOLR_HOST: solr
+      POSTGIS_PORT_5432_TCP_ADDR: pgbouncer
+      POSTGIS_DISABLE_SERVER_SIDE_CURSORS: "True"
+      HS_PATH: ${PWD}
+      PYTHONPATH: /hydroshare
+      DJANGO_SETTINGS_MODULE: hydroshare.settings
+      COLUMNS: 80
+      MC_HOST_hydroshare: "http://minioadmin:minioadmin@minio:9000"
+    volumes:
+      - ".:/hydroshare"
```

Comment:
perhaps the Dockerfile should copy the codebase into the image? I'm just thinking that eventually this image is going to need to be deployed into K8s